EntityManager/src/main/java/eu/neclab/ngsildbroker/entityhandler/controller/EntityBatchController.java

```diff
@@ -184,6 +187,7 @@ public Uni<RestResponse<Object>> upsertMultiple(HttpServerRequest request,
 	public Uni<RestResponse<Object>> appendMultiple(HttpServerRequest request,
 			List<Map<String, Object>> compactedEntities, @QueryParam(value = "options") String options,
 			@QueryParam("localOnly") boolean localOnly) {
+		boolean isNoOverwrite = options != null && options.contains("noOverwrite");
```

Comment:
Take it  from NGSIConstants class
public static final String NO_OVERWRITE_OPTION = "noOverwrite";

Reply:
done